manifests/s/ShaulEizikovich/vJoyDeviceDriver/2.1.9.1/ShaulEizikovich.vJoyDeviceDriver.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ShaulEizikovich.vJoyDeviceDriver
+PackageVersion: 2.1.9.1
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.sourceforge.net/project/vjoystick/Beta%202.x/2.1.9.1-160719/vJoySetup.exe
+  InstallerSha256: F103CED4E7FF7CCB49C8415A542C56768ED4DA4FEA252B8F4FFDAC343074654A
+  InstallerSwitches:
+    Silent: /VERYSILENT

```

```suggestion
    Silent: /VERYSILENT
    SilentWithProgress: /VERYSILENT
```
Winget runs in the `SilentWithProgress` mode by default, but when an application doesn't support progress, the silent switch should be added here as well